src/config/wallets.ts

```diff
@@ -1,16 +1,18 @@
 import * as React from 'react';
```

Comment:
@peterslany @danielsimao do either of you know why we have both `config/wallets` and `constants/wallets`? As far as I can tell everything in `config/wallets` is redundant apart from the `SELECTED_ACCOUNT_LOCAL_STORAGE_KEY` const which is used by the substrate provider.

We should be able to move this to a file in the constants directory and delete this file, but want to check with both of you before doing that in case there's a reason why we shouldn't.